lib/view_component/test_helpers.rb

```diff
@@ -20,7 +20,7 @@ def refute_component_rendered
     attr_reader :rendered_component
 
     def render_inline(component, **args, &block)
-      @rendered_component = controller.view_context.render(component, args, &block)
+      @rendered_component = controller.view_context.render_component(component, &block)
```

Comment:
```suggestion
      @rendered_component = 
        if if Rails.version.to_f >= 6.1
          controller.view_context.render(component, &block)
        else
          controller.view_context.render_component(component, &block)
        end
```

Reply:
Wouldn't it be better to check `Rails::VERSION::MAJOR >= 6 && Rails::VERSION::MINOR >= 1`?